content/aips/E96287A0963D-AIP-ADD-TERRA-USD-(UST)-TO-AAVE-V2.md

```diff
@@ -0,0 +1,129 @@
+ 
+The Wormhole Network underpins the bridge with a validator set that observes the chains it’s connected on. The Network is governed by 19 guardians that watch connected chains and sign message observations. Users deposit funds onto the token bridge from where their tokens are originating (e.g. UST on Terra) and the action is deconstructed into a message which is sent to the Wormhole Network who validates it and produces a signed message. The signed message is sent to the output token bridge (e.g. USTw on Ethereum) which verifies it and sends the user the wrapped bridge tokens.
+​
+Given the track record of UST, we believe the addition of UST would pose low risk to the Aave ecosystem.
+​
+6. Market data (Market Cap, 24h Volume, Volatility, Exchanges, Maturity)
+​
+Market cap: $8.5B total
+24 hour volume: $100M+
+Volatility: Link to volatility data: https://governance.aave.com/t/add-support-for-terra-usd-ust/6003/19
+Exchanges: Curve, Sushiswap, Uniswap, Balancer, Kucoin, Bittrex, Terraswap, Coinbase, MEXC, Gemini, Bitfinex
+​
+9. Social channels data (Size of communities, activity on Github)
+​
+Communities: 240k followers on Twitter, 25k in Telegram, 16k in Discord
+​
+10. Contracts date of deployments, number of transactions, number of holders for tokens
+	
+	Contracts date of deployments, number of transactions, number of holders for tokens
+	No. of transactions: 417,173
+	No. of holders: 21,707
+​
+## Rationale
+To mitigate concerns, a 0% collateral factor will allow UST to be supplied or borrowed - not used as collateral. This limits the risk for Aave and its users in other markets. As the wormhole contracts continue to be used and UST gains further adoption, the collateral factor can be increased. The use of rateStrategyStableTwo, reserve factor, and disabled stable borrowing is similar to other stablecoins per: https://docs.aave.com/risk/asset-risk/risk-parameters.
+​
+## Implementation
+​
+The UST price oracle will be served via [ChainLink](https://chain.link/), which ???includes??? the [UST/ETH feed](https://docs.chain.link/docs/ethereum-addresses).
+​
+UST will only be used for depositing, not as collateral. The following params are proposed:
+​
+​
+ - Strategy: rateStrategyStableTwo
+ - Base LTV As Collateral: 0
+ - Liquidation Threshold: 0
+ - Liquidation Bonus: 0
+ - Borrowing Enabled: true
+ - Stable BorrowRate Enabled: false
+ - Reserve Decimals: 6
+ - Reserve Factor: 2000
+​
+Following the steps from the [Aave governance docs](https://docs.aave.com/developers/protocol-governance/governance/propose-your-token-as-new-aave-asset), the following contracts were generated:
+​
+- Interest bearing aUST: 0x6C35faC782Be2Db776cdF777196a47916806892b
+- Variable Debt variableDebtUST: 0xd8aBEdc9CE320752e77C4A8706E64212EdE834B9
+- Stable Debt stableDebtUST: 0x107ed1832647436a0D39F09e65f4aA920242C0BB
+- Strategy Implementation for UST: 0x0dEDCaE8Eb22A2EFB597aBde1834173C47Cff186
+- Incentive Controller: 0x036cA61C1977c6EA52222Db81F725fDBd70eFBd7
```

Comment:
Please add additional comments about why an incentives controller is needed, if there will be rewards or there is a plan for it